tv.plex.PlexDesktop.yml

```diff
@@ -115,3 +117,12 @@ modules:
       - type: archive
         url: https://artifacts.plex.tv/plex-desktop-stable/1.80.3.4008-03daab12/linux/Plex-1.80.3.4008-03daab12-linux-x86_64.tar.bz2
         sha256: 8262b39c94bc20ef99daa9b786007c73166e9bf735af07e61ff1f352af4a601a
+    cleanup:
+      - /lib/dri
+      - /lib/libEGL.so*
+      - /lib/libdrm.so*
+      - /lib/libdrm_*.so*
+      - /lib/libpciaccess.so*
+      - /lib/libswresample.so*
```

Comment:
These two does not seem necessary?

Reply:
just removed all exact dups (by full name) so this package can benefit from runtime updates too.

It is true those lines might break future 24.08 update build, but maybe not.

what is your preferable opinion, shall I keep those lines or not?

Reply:
These two are not present in the runtime, they are probably needed for something. Let's not clean them up.

Reply:
I added those because I found them with the same name in /usr/lib in freedesktop runtime 23.08.

Reply:
Ah indeed, looks like my search failed